### PR TITLE
Show build badge in readme

### PR DESCRIPTION
**Pull request by marusak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+Live FAF instances:
+
+[https://retrace.fedoraproject.org/faf](https://retrace.fedoraproject.org/faf)
+
+FAF is part of the [ABRT project](http://github.com/abrt)
+
+### How it works
+
+Currently, typical crash reporting workflow consists of generating so called
```
Comment (xsuchy) [inline README.md]: so-called

File: README.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+[ureport](http://abrt.readthedocs.org/en/latest/ureport.html#ureport)
+(micro report) designed to be completely anonymous so it can be sent
+and processed automatically avoiding costly bugzilla queries and manpower.
```
Comment (xsuchy) [inline README.md]: Bugzilla

Comment (xsuchy) [reply README.md]: And the same in the rest of the document.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+FAF in this scenario works like a first line of defense — collecting
```
Comment (xsuchy) [inline README.md]: the first

File: README.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+massive amounts of similar reports and responding with tracker URLs
+in case of known problems.
+
+If user is lucky enough to hit a unique issue not known by FAF,
```
Comment (xsuchy) [inline README.md]: a user

File: README.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+reporting chain continues with reporting to bugzilla, more complex process
+which requires user having a bugzilla account and going through numerous steps
+including verification that the report doesn't contain sensitive data.
+
+You can read more about technical aspects of ABRT at our documentation site:
```
Comment (xsuchy) [inline README.md]: the technical

File: README.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+
+ * Support for various programming languages and software projects:
+   * C/C++
+   * Java
+   * Python
+   * Python 3
+   * Linux (kernel oops)
+ * De-duplication of incoming reports
+ * Clustering of similar reports (Problems)
+ * Collection of various statistics about platform
```
Comment (xsuchy) [inline README.md]: a platform

**Comment by marusak** [comment]
Updated @xsuchy

**Pull request merged**